### PR TITLE
Migrate devtools wiki & update old flutter wiki links

### PR DESCRIPTION
**Pull request by Piinks**
Part of https://github.com/flutter/flutter/issues/145009

This migrates the home and triage page of the devtools wiki into checked in files, and updates all old references to flutter/flutter/wiki

I'll update the original wiki pages to point to these new places after this lands

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read the [Flutter Style Guide] _recently_, and have followed its advice.
- [x] I signed the [CLA].
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or there is a reason for not adding tests.

![build.yaml badge]

If you need help, consider asking for help on [Discord].

[Contributor Guide]: https://github.com/flutter/devtools/blob/master/CONTRIBUTING.md
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[CLA]: https://cla.developers.google.com/
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[build.yaml badge]: https://github.com/flutter/devtools/actions/workflows/build.yaml/badge.svg

### CODE REVIEW

File: TRIAGE.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+### Label/project the issue:
+* Add labels for its proper category or categories ( “Inspector page”, “debugger page”, “bug”, etc.)
+* Add label “waiting for customer response” if you requested more details from reporter
+* Add label “fix it friday” if the issue should be fixed and looks easy to fix
+* Add to project [go/dart-devtools-ux-issues](https://github.com/orgs/flutter/projects/54/settings) if the issue is cross-screen issue
+
+### Prioritize the issue. 
+
+Follow the prioritization rubric [here](https://github.com/flutter/flutter/wiki/Issue-hygiene#priorities).
```
[the remark this answers is not in the document]
Comment (Piinks) [reply TRIAGE.md:23]: Oh thank you! I missed that one

File: packages/devtools_app/lib/src/framework/about_dialog.dart [hunk 2]
```diff
@@ -114,7 +114,8 @@ class _ContributingLink extends StatelessWidget {
 class _DiscordLink extends StatelessWidget {
   const _DiscordLink();
 
-  static const _discordWikiUrl = 'https://github.com/flutter/flutter/wiki/Chat';
+  static const _discordWikiUrl =
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/src/framework/about_dialog.dart:117]: can we remove "wiki" from the variable name?

File: tool/RELEASE_INSTRUCTIONS.md [hunk 3]
```diff
@@ -370,7 +370,7 @@ onto the `flutter/devtools` protected branch (`master`).
 
 ### Additional resources
 - `dart-lang/sdk` cherry-pick [Wiki](https://github.com/dart-lang/sdk/wiki/Cherry-picks-to-a-release-channel)
-- Flutter cherry-pick [Wiki](https://github.com/flutter/flutter/wiki/Flutter-Cherrypick-Process)
+- Flutter cherry-pick [Wiki](https://github.com/flutter/flutter/blob/master/docs/releases/Flutter-Cherrypick-Process.md)
```
Comment (kenzieschmoll) [inline tool/RELEASE_INSTRUCTIONS.md:373]: Remove references to the work "wiki" in the link titles

File: docs/README.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+## Home
```
Comment (kenzieschmoll) [inline docs/README.md:1]: we can delete this file all together

**Review by kenzieschmoll — APPROVED** [review]
lgtm with one comment

**Review by stuartmorgan — APPROVED** [review]
LGTM

File: docs/README.md (hunk 4 above)
Comment (Piinks) [reply docs/README.md:1]: @kenzieschmoll then should the docs/ directory be deleted then? This is the only thing in it now.
When we discussed the file structure in [flutter.dev/go/migrate-flutter-wiki-spreadsheet](https://docs.google.com/spreadsheets/d/1x65189ZBdNiLRygpUYoU08pwvXD4M-Z157c6pm8deGI/edit?usp=sharing) , it sounded like you all would self-organize other docs after the wiki was migrated?

Comment (kenzieschmoll) [reply docs/README.md:1]: Either way is fine. I'm happy to move things around later if you want to just land this as is 👍

**Pull request merged**